packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3233,8 +3233,8 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
 
   /// The configuration for this element.
   @override
-  Widget get widget => _widget!;
-  Widget? _widget;
+  Widget get widget => _widget;
+  Widget _widget = const _NullWidget();
```

Comment:
Maybe pull this out into a separate PR so we get separate benchmark runs for these two independent changes?

Reply:
Will do

Reply:
Removed